diff --git a/cmd/img_mgmt/include/img_mgmt/img_mgmt_config.h b/cmd/img_mgmt/include/img_mgmt/img_mgmt_config.h
index 6f781aa..a8e99e3 100644
--- a/cmd/img_mgmt/include/img_mgmt/img_mgmt_config.h
+++ b/cmd/img_mgmt/include/img_mgmt/img_mgmt_config.h
@@ -54,10 +54,8 @@
 
 #else
 
-/* No direct support for this OS.  The application needs to define the above
- * settings itself.
- */
-#error "Unknown OS/missing configuration"
+#define IMG_MGMT_UL_CHUNK_SIZE  512
+#define IMG_MGMT_BOOT_CURR_SLOT 0
 
 #endif
 
